### PR TITLE
Added CPS Counter preset and add support for right click

### PR DESCRIPTION
**Pull request by panpanpro888**
## Type of change

- [ ] Bug fix
- [x] New feature

## Description

Added a CPS counter preset, and will make the cps counter subscribe to the MouseButtonEvent, instead of the AttackEvent to also support right click.

## Related issues

#4064

# How Has This Been Tested?

Tested if the preset has the correct output.

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have added comments to my code in more complex areas.
- [x] I have tested the code in both development and production environments.

### CODE REVIEW

**Comment by panpanpro888** [comment]
Saw the issue #4064 also gonna add the fix for it here.

File: src/main/java/meteordevelopment/meteorclient/utils/misc/CPSUtils.java [hunk 1]
```diff
@@ -6,45 +6,71 @@
...
+    @EventHandler
+    public static void onClick(MouseButtonEvent event) {
+        if (event.button == 0 && event.action == KeyAction.Press) {
+            CPSUtils.clicks++;
+            CPSUtils.cps++;
+        } else if (event.button == 1 && event.action == KeyAction.Press) {
+            CPSUtils.rightClicks++;
+            CPSUtils.rightCps++;
+        }
+    }
```
Comment (Fiz-Victor) [inline src/main/java/meteordevelopment/meteorclient/utils/misc/CPSUtils.java:41]: Maybe do a mc.currentScreen != null check?

**Comment by BDARmech** [comment]
##<So close.>

**Comment by panpanpro888** [comment]
Maybe review the code now?

**Comment by panpanpro888** [comment]
@Fiz-Victor

**Comment by Fiz-Victor** [comment]
Looks fine

**Comment by Fiz-Victor** [comment]
why close?

**Pull request closed without merging**